src/main/java/com/wakeUpTogetUp/togetUp/api/alarm/AlarmService.java

```diff
@@ -34,7 +34,7 @@ public class AlarmService {
     private final MissionObjectRepository missionObjectRepository;
 
     @Transactional
-    public Alarm createAlarm(Integer userId, PostAlarmReq postAlarmReq) {
+    public Alarm createAlarmDeprecated(Integer userId, PostAlarmReq postAlarmReq) {
```

Comment:
P3: deprecated하니까 생각났는데 java에는 해당 메서드나 클래스에 달면 컴파일 시점에 경고 메시지를 주는 `@Deprecated` 어노테이션이 있다고 합니다.

해당 어노테이션을 도입하는 것에 대해 어떻게 생각하시나요?

``` java
public class Worker {
    /**
     * Calculate period between versions
     * @deprecated
     * This method is no longer acceptable to compute time between versions.
     * <p> Use {@link Utils#calculatePeriod(Machine)} instead.
     *
     * @param machine instance
     * @return computed time
     */
    @Deprecated(since = "4.5", forRemoval = true)
    public int calculate(Machine machine) {
        return machine.exportVersions().size() * 10;
    }
}
```

개인적인 생각으로는 위의 주석은 달지 않고 어노테이션 속성을 이용해서 표현하는 방향으로 생각하고 있습니다.

[참고]
https://inseok9068.github.io/springboot/springboot-deprecated/
https://www.baeldung.com/java-deprecated

Reply:
좋아요! 어노테이션으로 예전 것을 못쓰고 막고 함수명 중복을 위해 deprecated를 붙이는 것으로 할까요?

Reply:
> 예전 것을 못쓰고 막고 함수명 중복을 위해 deprecated를 붙이는 것으로 할까요?

이해가 잘 안됐는데 풀어서 설명해주실 수 있을까요?
@deprecated를 쓰면 함수명 중복이 가능한가요?

Reply:
어노테이션으로 예전 것을 못쓰게 권장하고(컴파일 시점에 경고),함수명 중복을 위해 deprecated를 붙이는 것으로 할까요?
입니다!

함수이름 뒤에 deprecated 붙이면
새로운 버전은 createAlarm 함수명을 사용할 수 있어서요!